donkeycar/utilities/TrackSpeedPlanner/uv.lock

```diff
@@ -0,0 +1,32 @@
+version = 1
```

Comment:
This file should be obsolete as we have our donkey env already setup and managed as part of our donkey install in RPi.